app/tests/integration/nimbus/conftest.py

```diff
@@ -126,7 +126,7 @@ def experiment_url(base_url, default_data, slugify):
 
 @pytest.fixture
 def experiment_name(request):
-    return f"{request.node.name[:76]}-{str(uuid.uuid4())[:4]}"
+    return f"{request.node.name[:75]}-{str(uuid.uuid4())[:4]}"
```

Comment:
Oh I see what happened, we had a latent bug in this name formatting that this targeting config accidentally triggered because it has a long name.

Reply:
That's what was causing the timeout @ncalexan , unrelated to the actual targeting failure, which fails with a more specific message.

Reply:
Yeah I was thinking that but that might have obfuscated the bug since it would be trimming the salt at the end that we inject to prevent name collisions during test runs, so that might have inadvertently caused other confusing failures.